.github/workflows/status-translation.yml

```diff
@@ -0,0 +1,50 @@
+name: status-translation
+
+on:
+  pull_request:
+    branches:
+      - gh-pages
+    paths:
+      - 'en/**'
+      - 'de/**'
+      - 'es/**'
+      - 'fr/**'
+      - 'it/**'
+      - 'ja/**'
+      - 'ko/**'
+      - 'pt-br/**'
+      - 'ru/**'
+      - 'sk/**'
+      - 'th/**'
+      - 'tr/**'
+      - 'uk/**'
+      - 'uz/**'
+      - 'zh-cn/**'
+      - 'zh-tw/**'
+
+jobs:
+  verify-translation:
+    runs-on: ubuntu-latest
+    permissions:
+       contents: write
+       pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+        
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+      - run: | 
+          npm install
```

Comment:
Why do we need to install the dependencies?

Reply:
Thanks for the feedback, it really doesn't make sense, I must have added it for testing and forgot to remove it